monorepo/website/src/pages/news/2025-11-17-november-update.mdx

```diff
@@ -0,0 +1,123 @@
+---
+layout: ../../layouts/NewsLayout.astro
+title: "Pathoplexus November Update"
+---
+
+<div>
+    <a href="/news">News</a> > <b>Pathoplexus November Update</b>
+</div>
+
+# Pathoplexus November Update
+
+*By the Pathoplexus Team - 17 November 2025*
+
+Since our last update in August, Pathoplexus has continued to grow - both in data and in community. Over the past two months, thousands of new sequences have been shared, several new features have gone live, and new members have joined our global community! In addition, we’re excited to be part of the upcoming joint PHA4GE & IPSN Conference in just a few weeks - read on to find out more!
```

Comment:
```suggestion
Since our last update in August, Pathoplexus has continued to grow - both in data and in community. Over the past two months, thousands of new sequences have been shared, several new features have gone live, and new members have joined our global community! In addition, we were excited to be part of the joint PHA4GE & IPSN Conference - read on to find out more!
```